Tests/Reference/global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+        "version": "3.0.100"
```

Comment:
Is this needed? If so, we'd need to make the same change to the release build image and test it still works (we had problems with a .net sdk upgrade for the release build in the past)

Reply:
yah - it's not needed, I've deleted it